### ISSUE TITLE
Step Into steps over in 1.9

### ISSUE THREAD

**Issue by NuSkooler** [Microsoft/vscode#19936]
- VSCode Version: 1.9.0 / 27240e71ef390bf2d66307e677c2a333cebf75af
- OS Version: Elementary OS Freya (0.3.2, Ubuntu 14.04 LTS)
`Linux nu-freya 3.16.0-77-generic #99~14.04.1-Ubuntu SMP Tue Jun 28 19:17:10 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`
- Node.js 6.5.0

Steps to Reproduce:

1. Place a breakpoint on a function call (local to module or within project scope doesn't seem to matter)
2. Press Step Into (F11) or click Step Into in the UI

Expected: Step into the method.
Results: Steps over (as if F10 was pressed)

**Comment by weinand** [comment]
@NuSkooler please try our new node debugger by setting `"type"` to `"node2"`.

**Comment by NuSkooler** [comment]
@weinand That does the trick! Is `node` simply deprecated?

**Comment by weinand** [comment]
"node2" is for node.js versions that support the new "--inspect" option (version >= 6.3).
"node" is for all other node versions.

**Comment by nicksnyder** [comment]
@weinand I tried updating the "Unit Tests" target to "node2" but I get this error:

```
Inspector support is not available with this Node.js build
/Users/nick/code/vscode/.build/electron/Code - OSS.app/Contents/MacOS/Electron: bad option: --inspect=28539
```

My node version is >= 6.3 though.
```
$ node --version
v6.9.5
```

**Comment by weinand** [comment]
@nicksnyder Electron does not (yet) support the new 'inspector' protocol that 'node2' uses. So for the time being you'll have to continue to use 'node'.

**Comment by nicksnyder** [comment]
@weinand so I will be unable to step into functions while debugging unit tests until electron supports the node2 protocol? Can I downgrade my version of node to workaround?

**Comment by weinand** [comment]
@nicksnyder you are seeing the "Step Into" problem with Electron? What is the version of the node runtime builtin into your Electron?
What is your platform?

Since no one in our team was able to reproduce this problem, we do not know in what node version this problem started to occur.
You can only downgrade node by downgrading Electron since you seem to use the node built into Electron.

**Comment by nicksnyder** [comment]
> you are seeing the "Step Into" problem with Electron?

Yes, I think that is correct (see #20776 for exact details).

> What is your platform?

macOS Sierra 10.12.2

> What is the version of the node runtime builtin into your Electron?

How do I figure this out? The version of Electron is v1.4.6. I haven't done anything special, just checked out the code and trying to run unit tests.

```
$ otool -L "./Code - OSS.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libnode.dylib"
./Code - OSS.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libnode.dylib:
	/usr/local/lib/libnode.dylib (compatibility version 0.0.0, current version 0.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1153.18.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1213.0.0)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 120.0.0)
```

**Comment by weinand** [comment]
@nicksnyder your problem is a dup of https://github.com/Microsoft/vscode/issues/23494 which has been fixed in 1.11.0
